tests/default/_core/search/query/terms.yaml

```diff
@@ -114,6 +140,62 @@ chapters:
                 title: Beauty and the Beast
                 genre: adventure
 
+  - synopsis: Search using TermsQuery with an array of terms with boost.
```

Comment:
Let's split up this `terms.yaml` into `terms/array_with_boost.yaml` and others? Otherwise we have a lot of unrelated setup/teardown and we'd like to use these in documentation examples.

Reply:
done